pkg/api/error.go

```diff
@@ -7,6 +7,12 @@ var (
 	// ErrAPIDefinitionNotFound is used when the api was not found in the datastore
 	ErrAPIDefinitionNotFound = errors.New(http.StatusNotFound, "api definition not found")
 
+	// ErrAPINameExists is used when the API name is already registeres on the datastore
+	ErrAPINameExists = errors.New(http.StatusBadRequest, "api name is already registered")
+
+	// ErrAPIListenPathExists is used when the API name is already registeres on the datastore
```

Comment:
ErrAPIListenPathExists is used when the API ~~name~~ listen path is already registeres on the datastore